src/components/conference/audio.jsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+const Audio = (props) => {
+  const { track } = props;
+  if (track.isLocal()) {
```

Comment:
Do we know what isLocal does? The JitsiTrack API says the isLocal method should just throw an error upon being called. This is weird...